.github/workflows/CI.yml

```diff
@@ -0,0 +1,94 @@
+name: CI
+# Run on master, tags, or any pull request
+on:
+  schedule:
+    - cron: '0 2 * * *'  # Daily at 2 AM UTC (8 PM CST)
+  push:
+    branches: [master]
+    tags: ["*"]
+  pull_request:
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "1.5"  # Invenia Prod version
```

Comment:
Testing on 1.0 complained about a lot of stuff so I didn't think there would be much gain in testing on it.

I feel these are sufficient?

Reply:
The Project.toml should bump the existing julia compat: `julia = "1"` if this package doesn't run on 1.0